src/and-cli-dotnet.test.ts

```diff
@@ -71,6 +72,38 @@ describe("and-cli-dotnet", () => {
     shouldDisplayHelpMenu("dotnet");
 
     // #endregion help
+
+    // -----------------------------------------------------------------------------------------
+    // #region publish
+    // -----------------------------------------------------------------------------------------
+
+    givenOptions(DotnetPublish.getOptions().toArray(), (option: string) => {
+        describe("when no solution can be found", () =>
+            shouldDisplayError(
```

Comment:
Curious if we're not concerning ourselves with calling out all three A aspects (Arrange, Act, and Assert).  I would expect that to be associated with this `shouldDisplayError` method if so.

Reply:
Yeah, I'm not too sure either. I put the Arrange, Act and Assert steps inside of that function, which I assume most people would just follow if they wanted to see what deeper setup/action is being performed - is there any extra context that putting all three aspects here would provide?